edenia/antelope-tools
# Feat/add tooltips to cards

### Add tooltips to cards to provide more info

### What does this PR do?

- Add tooltips in the main  stress test and EVM dashboard pages
- Resolve #1281
- Resolve #1284
- Resolve #1285
- Fix accessibility errors: image elements with redundant alt texts, links without recognizable names and lists don't contain only \<li\> elements

### Steps to test

1. Run the project locally
2. Go to main page
3. Check the tooltip appers when hover over the card with the info icon

### Screenshots

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/8c34754a-4d0f-4da0-bba2-30a081ac6a36)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/ee965e7c-b588-4563-aa99-c6f0e86a59a1)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/c9fef338-fcef-4291-867e-709fe96ca7f6)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/6cec04e7-25e1-4ed5-8691-ab95e476318a)
